addon/components/polaris-color-picker.js

```diff
@@ -88,7 +94,7 @@ export default Component.extend({
   },
 
   actions: {
-    draggerMoved({x, y}) {
+    draggerMoved({ x, y }) {
```

Comment:
Why on earth has this thing added spaces to just this one place, and removed them from pretty much everywhere else?!

Reply:
^^ @sivakumar-kailasam can I get an answer on this pls? 😅

Reply:
that's prettier doing this ([example](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAzArlMMCW0AEAFnADYkQAUwAHgDQCeAvgJT7CMi0gQAOu0AZ2SgAhgCcxEAO4AFcQiEoRANwg4AJpxAAjMSLABrODADKPfTigBzZDDHo4XSwLhiYMvVYC2I5KhEkLlwAVgLUAEJ6hsYmIl5wADKWcH4BQSDmYi5iyDoi2vTkUFo8YpYwAOoaMITIABwADFylEC4Vejy5pXDZyilcYnAAjug4gx4i3r5I-oGOIC5eOLb28wKWViRwAIroEPCpc1ww+VXqNcgATMd6OCQbAMIQXj65UND9IOguACr5irMXIxGEA)). So out of our control rn 😅